### ISSUE TITLE
Is there a plan to add Firetools to official Fedora repositories ?

### ISSUE THREAD

**Issue by Nokia808** [netblue30/firetools#41]
Recently - after long waiting time - Firejail become available in Fedora repositories ....... I'm not know if the package of Firejail is official from you or pre-built by 3rd party (maintainer). If it is not official from you, so this issue is non sense & please close it. But, if it is official package from you, then I would like to ask if there a plan to add Firetools to official Fedora package or not ?

**Comment by startx2017** [comment]
Not that we know of. Usually Fedora is very slow to pick up new programs.

**Comment by MystesofEternity** [comment]
Agreed, even my favorite text editor Scite is still not in the fedora-30 repo yet it was in the fedora-29 repo

**Comment by neufeind** [comment]
firejail itself is available in Fedora meanwhile. Unfortunately firetools not yet. But there is at least a copr-repository that builds for all Fedora-releases. Maybe that hints that firetools might be on a way to inclusion as well?
https://copr.fedorainfracloud.org/coprs/nielsenb/firejail/
